### PR TITLE
chore(components/google-cloud): Create Release Branch for Google cloud pipeline components 0.1.3

### PR DESCRIPTION
**Pull request by sasha-gitg**
Releasing Google cloud pipeline components 0.1.3 which Includes:

*   Use correct dataset type when passing dataset to CustomTraining.
*   Bump google-cloud-aiplatform to 1.1.1

### CODE REVIEW

File: .readthedocs.yml [hunk 1]
```diff
@@ -1,8 +1,9 @@
 # See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
 version: 2
 sphinx:
-  configuration: docs/conf.py
+  configuration: components/google-cloud/docs/source/conf.py
```
Comment (chensun) [inline .readthedocs.yml:4]: This seems to be an unintended change?

Comment (sasha-gitg) [reply .readthedocs.yml:4]: This is to temporarily work around this issue: https://github.com/kubeflow/pipelines/pull/5887/files#diff-cde814ef2f549dc093f5b8fc533b7e8f47e7b32a8081e0760e57d5c25a1139d9L4

Comment (chensun) [reply .readthedocs.yml:4]: I see. Thanks for the pointer.

**Pull request merged**